api/app/settings/common.py

```diff
@@ -942,3 +942,81 @@
 FLAGSMITH_ON_FLAGSMITH_SERVER_API_URL = env(
     "FLAGSMITH_ON_FLAGSMITH_SERVER_API_URL", default=FLAGSMITH_ON_FLAGSMITH_API_URL
 )
+
+# LDAP setting
+LDAP_INSTALLED = importlib.util.find_spec("ldap")
```

Comment:
I think it's better to search for our module here.

Reply:
I'm not sure what you mean here - `ldap` is our own module.

Reply:
python-ldap has the same module name so I got confused, apologies. Maybe we should rename our module to flagmsith-ldap.

Reply:
Yeah, that's probably a good shout! I'll do that.

Reply:
This is done [here](https://github.com/Flagsmith/flagsmith/pull/2923/commits/87ab03393bc3b053134d318211e6f36c5f447058)